source/slang/slang-check-decl.cpp

```diff
@@ -2779,11 +2779,7 @@ namespace Slang
         //      subscript(uint index) -> T { get; }
         //
 
-        bool anyAccessors = false;
-        for(auto accessorDecl : decl->getMembersOfType<AccessorDecl>())
-        {
-            anyAccessors = true;
-        }
+        bool anyAccessors = decl->getMembersOfType<AccessorDecl>().hasContent();
```

Comment:
Not sure if it was added in this change, but `hasContent` is a slightly odd function name. I would have expected `!someList.isEmpty()` to be the more natural way to express that idea.